lib/order.rb

```diff
@@ -0,0 +1,26 @@
+require_relative 'menu.rb'
+class Order
+  attr_reader :basket, :menu
+  def initialize
+    @basket = Hash.new(0)
+    @menu = Menu.new
+
+  end
+
+  def view_basket
+    @basket
+  end
+
+  def add_dish(dish, number)
+    # fail "We're out of #{dish}" unless menu.dish?(dish)
+    @basket[dish] += number
```

Comment:
A very clever method to add to the basket hash! I was confused at first because what if the dish doesn't exist yet how would it append the number to the value pair? But this is smart! It creates the new value pair 'dish => number' if the dish key doesn't exist yet and if it does and the user wants to add more of that dish to the basket it will append on the number of times they want it again!

Reply:
Interesting! This is a clever method that solves the typical scenario of a group of friends ordering food: "Let's order 1 🍕 , no... let's order 2, not..., let's make it 3!" The final value of items to be ordered will be updated based on the last request. This is all fine.

However, take a look at your menu (`@menu_items` under your Menu class). What would happen if I try to order a salad for example? e.g. `add_dish("salad", 1)`. What would happen after I run that method with these values on an instance of the Order Class?